typings/tsconfig.json

```diff
@@ -12,6 +12,9 @@
   "references": [
     {
       "path": "./environment/tsconfig.json"
+    },
+    {
+      "path": "./static-assets/tsconfig.json"
```

Comment:
```suggestion
```

As we agreed in offline to perform TS conversion separately, let's also restore this.